### DISCUSSION TITLE
Is it possible to add chrome extension to the tool?

### DISCUSSION THREAD

**Discussion by thuandohoang** [jhen0409/react-native-debugger#803] [Q&A]
Firstly, thanks for the awesome tool. I'm using GraphQL, and the browser network is not ideal for this technology, all API endpoints are `/query`, difficult for inspect the network.

There is a well know extension called [graphql-network-inspector](https://chromewebstore.google.com/detail/graphql-network-inspector/ndlbedplllcgconngcnfmkadhokfaaln). That'd be great if I can use this extension inside react-native-debugger.

**Comment by m3hari** [comment]
Yes please
